jobs/release.Jenkinsfile

```diff
@@ -298,19 +298,25 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                     --acl=${acl} ${s3_stream_dir}/builds
                 """)
 
-                if (!pipecfg.hacks?.skip_plume_release_task) {
-                    // Run plume to publish official builds; This will handle modifying
-                    // object ACLs, modifying AMI image attributes,
-                    // and creating/modifying the releases.json metadata index
-                    // Note here we pass the bucket instead of `s3_stream_dir` but
-                    // plume currently only actually interacts with objects under
-                    // the `s3_stream_dir` key (i.e. `pipecfg.s3.builds_key`).
-                    // We'll make this more explicit in the future.
+                def bucket_prefix = "${pipecfg.s3.bucket}/${pipecfg.s3.builds_key}"
+
+                // make AMIs public if not already the case
+                if (!pipecfg.clouds?.aws?.public) {
+                    shwrap("""
+                    cosa shell -- plume make-amis-public \
+                        --version ${params.VERSION} \
+                        --stream ${params.STREAM} \
+                        --bucket-prefix ${bucket_prefix} \
```

Comment:
I still need to review the COSA PR, but just from scrolling this it seems odd that a call to make AMIs public needs info about an s3 bucket.

Reply:
It reads the release metadata directly from the bucket. `plume` commands traditionally are not "cosa workdir-aware", so I'd prefer not to have it start looking for a `builds/` dir there. But what we could do is hand it the release metadata file directly ourselves. That should actually allow us to drop the `--version` and `--stream` switches too.

I'll see about enhancing it for that, but will do that as a follow-up to https://github.com/coreos/coreos-assembler/pull/3240 so that we can get rid of `plume release` first to not support the old way anymore.